src/utils/hashPass.js

```diff
@@ -0,0 +1,44 @@
+const bcrypt = require('bcrypt');
+const { saltRounds } = require('../config/index');
+const Password = {
+  hash: async (password) => {
+    try {
+      const genSalt = () => {
+        return new Promise((resolve, reject) => {
+          bcrypt.genSalt(saltRounds, (err, salt) => {
+            if (err) {
+              reject(err);
+            }
+            resolve(salt);
+          });
+        });
+      }
+      const salt = await genSalt();
+
+      const hashedPassword = await new Promise((resolve, reject) => {
```

Comment:
don't make redundant variables, you can return here instead. return `new Promise((resol....`